spec/react_on_rails/assets_precompile_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative "spec_helper"
+require "react_on_rails/assets_precompile"
+
+# require "tmpdir"
+# require "tempfile"
+
```

Comment:
@samnang @mapreal19 What's the best way to write these tests given the dependencies on the Rails and ReactOnRails config values? I"m thinking of using tmpdir and tempfile, along with some sort of way to set the the global config values temporarily.

And if we make files for test in a tempdir, we don't need to cleanup, right?